### PR TITLE
Add support for separately marking brightcove videos annotations.

### PR DESCRIPTION
**Pull request by iulianp26**

### CODE REVIEW

File: annotations/cypher.go [hunk 1]
```diff
@@ -98,15 +104,21 @@ func (s service) Delete(contentUUID string) (bool, error) {
 
 	var deleteStatement string
 
-	if s.platformVersion == "v2" {
+	if s.platformVersion == v2PlatformVersion {
```
Comment (tamas-molnar) [inline annotations/cypher.go]: Consider using a switch statement.

File: annotations/cypher.go [hunk 2]
```diff
@@ -320,9 +332,14 @@ func dropAllAnnotationsQuery(contentUUID string, platformVersion string) *neoism
 	//WE STILL NEED THIS UNTIL EVERYTHNG HAS A LIFECYCLE PROPERTY!
 	// -> necessary for brands - which got written by content-api with isClassifiedBy relationship, and should not be deleted by annotations-rw
 	// -> so far brands are the only v2 concepts which have isClassifiedBy relationship; as soon as this changes: implementation needs to be updated
-	if platformVersion == "v2" {
```
Comment (tamas-molnar) [inline annotations/cypher.go]: Consider using a switch statement,

Comment (iulianp26) [inline annotations/cypher.go]: Changes done, refactored also the delete functions as per @BerniVarga comment.

**Comment by iulianp26** [comment]
Done additional refactoring for delete function according to @BerniVarga comment.

**Pull request merged**